python_src/src/lamp_py/postgres/postgres_utils.py

```diff
@@ -289,6 +289,7 @@ def truncate_table(
             truncate_query = f"{truncate_query} CASCADE"
 
         self.execute(sa.text(f"{truncate_query};"))
+        self.execute(sa.text(f"ANALYZE {truncat_as};"))
```

Comment:
Run `ANALYZE` after a truncate, I think this will speed up our Static temp table loading times.